doc/source/tune/user-guide.rst

```diff
@@ -346,7 +454,7 @@ Distributed Checkpointing
 
 On a multinode cluster, Tune automatically creates a copy of all trial checkpoints on the head node. This requires the Ray cluster to be started with the :ref:`cluster launcher <cluster-cloud>` and also requires rsync to be installed.
 
-Note that you must use the ``tune.checkpoint_dir`` API to trigger syncing.
+Note that you must use the ``tune.checkpoint_dir`` API to trigger syncing (or use a Ray built-in model type described here).
```

Comment:
Also want to link to that example `custom_func_checkpointing.rst` or a better one from github.